sphinx_rtd_theme/static/js/versions.js_t

```diff
@@ -123,3 +124,68 @@ function renderLanguages(config) {
       document.dispatchEvent(event);
     });
  });
+{%- endif %}
+
+
+{%- if theme_version_selector or theme_language_selector %}
+function onSelectorSwitch(event) {
+     const option = event.target.selectedIndex;
+     const item = event.target.options[option];
+     window.location.href = item.dataset.url;
+ }
+
+document.addEventListener("readthedocs-addons-data-ready", function(event) {
+  const config = event.detail.data();
+
+  const dropdownSelectors = document.querySelector("div.dropdown-selectors");
+
+  {%- if theme_version_selector %}
+  const versionSelect = `
+    <select id="readthedocs-version-selector">
+      ${ config.versions.active.map(
+          (version) => `
+          <option
+              value="${ version.slug }"
+              ${ config.versions.current.slug === version.slug ? 'selected="selected"' : '' }
+              data-url="${ version.urls.documentation }">
+              ${ version.slug }
+          </option>`
+      ).join("\n") }
+    </select>
+  `;
+
+  dropdownSelectors.insertAdjacentHTML("beforeend", versionSelect);
+  const versionSelector = document.querySelector("#readthedocs-version-selector");
+  versionSelector.addEventListener("change", onSelectorSwitch);
+  versionSelector.innerHTML = versionSelect;
+  {%- endif %}
+
+
+  {%- if theme_language_selector %}
+  if (config.projects.translations.length) {
+    // Add the current language to the options on the selector
+    let languages = config.projects.translations.concat(config.projects.current);
+    languages = languages.sort((a, b) => a.language.name.localeCompare(b.language.name));
+
+    const languageSelect= `
+      <select id="readthedocs-version-selector">
```

Comment:
Same note on id usage here:

```suggestion
      <select>
```

CSS selector is `div.switch-menus > div.version-switch > select` and has plenty of specificity.